src/Advection/vector_invariant_velocity_upwinding.jl

```diff
@@ -0,0 +1,97 @@
+const VectorInvariantVelocityVerticalUpwinding  = VectorInvariant{<:Any, <:Any, <:Any, <:Any, <:AbstractUpwindBiasedAdvectionScheme, <:VelocityUpwinding}
+
+#####
+##### Velocity upwinding is a Partial Upwinding where the upwind choice occurrs _inside_
+##### the difference operator (i.e velocity upwinding) instead of outside (i.e., derivative upwinding).
+##### _MOST_ stable formulation at the expense of a low kinetic energy
+##### 
+
+##### 
+##### Velocity Upwinding of Divergence flux
+#####
+
+@inline function Auᶜᶜᶜ(i, j, k, grid, scheme, u) 
```

Comment:
What does "Au" mean? Is this "Ax * u", or something different?

Reply:
Ax_uᶜᶜᶜ I'll change name